devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -193,11 +197,20 @@ def configure() {
 		        zigbee.temperatureConfig(30, 300) +
 		        zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 30, 21600, 0x10) +
 		        zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 30, 3600, 100, [destEndpoint: 0x02])
-	} else {
+	} else if (isFrientSensor()) {
+		return refresh() + 
+		        zigbee.configureReporting(zigbee.RELATIVE_HUMIDITY_CLUSTER, 0x0000, DataType.UINT16, 60, 600, 1*100) +
+		        zigbee.configureReporting(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000, DataType.INT16, 60, 600, 0xA) +
+		        zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020, DataType.UINT8, 30, 21600, 0x1)
+    } else {
 		return refresh() +
 		        zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0x104E]) +   // New firmware
 		        zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0xC2DF]) +   // Original firmware
 		        zigbee.batteryConfig() +
 		        zigbee.temperatureConfig(30, 300)
 	}
 }
+
+private Boolean isFrientSensor() {
+	device.getDataValue("manufacturer") == "Develco Products A/S"
```

Comment:
The manufacturer is "frient A/S" in Your fingerprint.
What's the correct version ?

Reply:
manufacturer name is modifed to "frient A/S"